smac/intensification/simple_intensifier.py

```diff
@@ -160,9 +130,11 @@ def get_next_run(self,
                      repeat_configs: bool = True,
                      ) -> typing.Tuple[RunInfoIntent, RunInfo]:
         """
-        Selects which challenger to use based on the iteration stage and set the iteration parameters.
+        Selects which challenger to use based on the iteration stage and set the
+        iteration parameters.
         First iteration will choose configurations from the ``chooser`` or input challengers,
-        while the later iterations pick top configurations from the previously selected challengers in that iteration
+        while the later iterations pick top configurations from the previously
```

Comment:
This still looks very much like the successive halving documentation.

Reply:
Updated it!